src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/LibraryImportGenerator.cs

```diff
@@ -61,6 +61,8 @@ public static class StepNames
 
         public void Initialize(IncrementalGeneratorInitializationContext context)
         {
+            context.CheckForAllowUnsafeBlocks(static () => Diagnostic.Create(GeneratorDiagnostics.RequiresAllowUnsafeBlocks, null));
```

Comment:
Doesn't every C# project in .NET 7 automatically reference this generator?

Reply:
Yes, that what I am realizing. I need to narrow the logic a great deal. blech.